src/envars.ts

```diff
@@ -98,6 +98,14 @@ export type EnvVars = {
 
 type EnvVarKey = keyof EnvVars;
 
+/**
+ * Provider env keys that are required to be set either in env or in config to interact with provider services.
+ */
+export enum ProvidersRequiringAPIKeysEnvKeys {
+  OPENAI_API_KEY = 'OPENAI_API_KEY',
+  ANTHROPIC_API_KEY = 'ANTHROPIC_API_KEY',
+}
+
```

Comment:
I don't think this belongs in this file. Could you move it to be co-located with the providers?

Reply:
I put this here given that it does relate to the env vars that are checked. Is there a specific place in providers you think this would better live and if so why there and not near the envvars keys they are meant to mirror?

Reply:
In my mind these are a property of the providers so they should live with/on the providers. Secondly, this has a side effect of defining which providers require an environment variable, which to me feels like a property of the provider as well. Lastly, the envars file is to manage environment variables, it shouldn't have any application logic in it.

Reply:
Understood -- moved!